### PR TITLE
Add FC093 for generated README text

### LINKED ISSUE #658: Catch TODO in autogenerated README
When we use `chef generate cookbook`, we get an autogenerated `README.md` containing this line:

```
TODO: Enter the cookbook description here.
```

Could we make a rule to catch this? Like there are rules to catch autogenerated lines in `metadata.rb`.

### PR DESCRIPTION
**Pull request by gordonbondon**
Should fix #658

### CODE REVIEW

**Comment by tas50** [comment]
Way cool. Thanks

**Comment by tas50** [comment]
One more thing actually. Can you open a PR to https://github.com/Foodcritic/foodcritic-site so we can document the new rule.

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -88,6 +88,10 @@ def metadata_file(*args, &block)
       file("metadata.rb", *args, &block)
     end
 
+    def readme_file(*args, &block)
```
Comment (coderanger) [inline spec/spec_helper.rb]: This might be overkill for a dedicated helper :)

**Comment by martinmosegaard** [comment]
Thanks @gordonbondon 👍

**Comment by gordonbondon** [comment]
@tas50 done - Foodcritic/foodcritic-site#43

**Pull request merged**